charts/incubator/passky-server/values.yaml

```diff
@@ -0,0 +1,103 @@
+image:
+  repository: rabbitcompany/passky-server
+  pullPolicy: IfNotPresent
+  tag: 8.1.6@sha256:ef803c3ffbabde6b79b996f5b33fcab0515f6439ccc961129d5da044c5518979
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+    runAsGroup: 0
+    runAsUser: 0
+service:
+  main:
+    ports:
+      main:
+        port: 18080
+        targetPort: 80
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          env:
+            SERVER_LOCATION: "US"
+            SERVER_CORES: 1
+            ADMIN_USERNAME: "admin"
+            ADMIN_PASSWORD: ""
+            CF_TURNSTILE_SITE_KEY: "1x00000000000000000000AA"
+            CF_TURNSTILE_SECRET_KEY: "1x0000000000000000000000000000000AA"
+            DATABASE_ENGINE: mysql
+            MYSQL_HOST:
+              secretKeyRef:
+                expandObjectName: false
+                name: '{{ printf "%s-%s" .Release.Name "mariadbcreds" }}'
+                key: plainhost
+            MYSQL_PORT: 3306
+            MYSQL_DATABASE: "{{ .Values.mariadb.mariadbDatabase }}"
+            MYSQL_USER: "{{ .Values.mariadb.mariadbUsername }}"
+            MYSQL_PASSWORD:
+              secretKeyRef:
+                expandObjectName: false
+                name: '{{ printf "%s-%s" .Release.Name "mariadbcreds" }}'
+                key: mariadb-password
+            MYSQL_SSL: "false"
+            MYSQL_SSL_CA: "/etc/ssl/certs/ca-certificates.crt"
+            REDIS_HOST:
+              secretKeyRef:
+                expandObjectName: false
+                name: '{{ printf "%s-%s" .Release.Name "rediscreds" }}'
+                key: plainhost
+            REDIS_PORT: 6379
+            REDIS_PASSWORD:
+              secretKeyRef:
+                expandObjectName: false
+                name: '{{ printf "%s-%s" .Release.Name "rediscreds" }}'
+                key: redis-password
+            REDIS_LOCAL_HOST:
+              secretKeyRef:
+                expandObjectName: false
+                name: '{{ printf "%s-%s" .Release.Name "rediscreds" }}'
+                key: plainhost
+            REDIS_LOCAL_PORT: 6379
+            REDIS_LOCAL_PASSWORD:
+              secretKeyRef:
+                expandObjectName: false
+                name: '{{ printf "%s-%s" .Release.Name "rediscreds" }}'
+                key: redis-password
+            MYSQL_CACHE_MODE: 0
```

Comment:
Please group mysql vars together, but why we disable cache mode? Is that default?

Reply:
Name MYSQL_CACHE_MODE is a little vague and doesn't explain what it actually do.

Passky Server has 4 mysql cache modes. All modes are only used to calculate total amount of users and passwords stored on the Passky Server.
 
mode 0 =  calculate **total** amount of users and passwords stored on the server **every time user call certain API endpoint**.

mode 1 = calculate **approximation of total** users and passwords stored on the server **every time user call certain API endpoint**.

mode 2 = calculate **total** amount of users and passwords stored on the server **only once per day**.

mode 3 = calculate **approximation of total** users and passwords stored on the server **only once per day**. 

Redis will still cache the result for few seconds in order to reduce the load if multiple users are spamming this API endpoint.

Mode 2 and 3 are recommended for instances with thousands of users.


Do you have an example on how to group mysql variables together?

Reply:
> Do you have an example on how to group mysql variables together?

Move it up with the other variables :D